controllers/reviews.controller.js

```diff
@@ -0,0 +1,12 @@
+const { selectReviewId } = require("../models/reviews.model");
+
+exports.getReviewId = (req, res, next) => {
+  const { review_id } = req.params;
+  selectReviewId(review_id)
+    .then((result) => {
+      res.status(200).send({ review: result });
+    })
+    .catch((err) => {
```

Comment:
You can simply pass next to the catch block here. There is no "if an error", if we get to the catch block something has errored and we need to send it back to our error handlers. By just adding .catch(next) that is all taken care of.